_scripts/emailApplierTemplates/mailValidation

```diff
@@ -0,0 +1,18 @@
+const capitalize = (s: string): string =>
+  s.charAt(0).toUpperCase() + s.slice(1);
+
+const splitAndCap = (s: string): string =>
+  s
+    .toLowerCase()
+    .split(" ")
+    .map(e => capitalize(e))
+    .join(" ");
```

Comment:
Could we include some tests for this functionality?

Reply:
I added two new snapshots in https://github.com/pagopa/io-functions-app/pull/304/commits/3475c7bffc249d3aba733e237285f511494dedf1